vcd_cli/user.py

```diff
@@ -0,0 +1,140 @@
+              help='User\'s telephone number.')
+@click.option('-i',
+              '--im',
+              required=False,
+              default='',
+              metavar='[im]',
+              help='User\'s im address.')
+@click.option('-E',
+              '--enabled',
+              is_flag=True,
+              required=False,
+              default=False,
+              metavar='[enabled]',
+              help='Enable user')
+@click.option('--alert-enabled',
+              is_flag=True,
+              required=False,
+              default=False,
+              metavar='[alert_enabled]',
+              help='Enable alerts')
+@click.option('--alert-email',
+              required=False,
+              default='',
+              metavar='[alert_email]',
+              help='Alert email address')
+@click.option('--alert-email-prefix',
+              required=False,
+              default='',
+              metavar='[alert_email_prefix]',
+              help='String to prepend to the alert message subject line')
+@click.option('--external',
+              is_flag=True,
+              required=False,
+              default=False,
+              metavar='[is_external]',
+              help='Indicates if user is imported from an external source')
+@click.option('--default-cached',
+              is_flag=True,
+              required=False,
+              default=False,
+              metavar='[default_cached]',
+              help='Indicates if user should be cached')
+@click.option('-g',
+              '--group-role',
+              is_flag=True,
+              required=False,
+              default=False,
+              metavar='[is_group_role]',
+              help='Indicates if the user has a group role')
+@click.option('-s',
+              '--stored-vm-quota',
+              required=False,
+              default=0,
+              metavar='[stored_vm_quota]',
+              help='Quota of vApps that this user can store')
+@click.option('-d',
+              '--deployed-vm-quota',
+              required=False,
+              default=0,
+              metavar='[deployed_vm_quota]',
+              help='Quota of vApps that this user can deploy concurrently')
+def create(ctx, user_name, password, role_name, full_name, description, email, telephone, im, enabled,
+           alert_enabled, alert_email, alert_email_prefix, external, default_cached, group_role,
+           stored_vm_quota, deployed_vm_quota):
+    try:
+        client = ctx.obj['client']
+        org_name = ctx.obj['profiles'].get('org_in_use')
+        in_use_org_href = ctx.obj['profiles'].get('org_href')
+        org = Org(client, in_use_org_href, org_name == 'System')
+        role = org.get_role(role_name)
+        if len(role) == 0:
+            raise Exception('Role \'%s\' does not exist.' % role_name)
+        else:
+            role_href = role[0]['href']
+            u = org.create_user(user_name, password, role_href, full_name, description, email, telephone, im,
```

Comment:
please make sure this passes flake8 validation

Reply:
checked. I don't see any failures.